js/_codemirror.ts

```diff
@@ -89,6 +92,7 @@ export const extensions = {
         oneDarkTheme,
         syntaxHighlighting(oneDarkHighlightStyle),
     ],
+    'vim': vim({ status: true }),
```

Comment:
Personally, I prefer the status bar but it could be disabled here. Here's what it looks like (at the bottom of the editor):

![Screen Shot 2022-08-05 at 7 01 23 PM](https://user-images.githubusercontent.com/11131775/183222237-071d727d-3e3c-4867-b070-7550c84729e8.png)